src/memray/_memray/source.cpp

```diff
@@ -63,6 +75,21 @@ FileSource::close()
     _close();
 }
 
+void
+FileSource::findReadableSize()
+{
```

Comment:
Can you add a comment here explaining what is the "readable size" for quick future reference?

Reply:
Good idea, will do.